Security/Firewall/OAuthListener.php

```diff
@@ -71,6 +71,7 @@ public function handle(GetResponseEvent $event)
         $token->setToken($oauthToken);
 
         try {
+            /** @var TokenInterface|Response $returnValue */
```

Comment:
`Response` is not possible here according to Symfony types, but is according to the code which follows, this just reinforces the code.